test/unit/com/unifina/service/SubscriptionServiceSpec.groovy

```diff
@@ -17,16 +17,28 @@ import spock.lang.Specification
 @Mock([FreeSubscription, IntegrationKey, PaidSubscription, Permission, Product, Stream, Subscription])
 class SubscriptionServiceSpec extends Specification {
 
-	SecUser user, user2
-	Stream s1, s2, s3
+	SecUser user, user2, user3
+	Stream s1, s2, s3, pub1Inbox, pub2Inbox, user3Inbox
 	Product product
 
 	void setup() {
 		user = new SecUser(username: "me@streamr.com").save(failOnError: true, validate: false)
 		user2 = new SecUser(username: "someoneElse@streamr.com").save(failOnError: true, validate: false)
+		user3 = new SecUser(username: "0x26e1ae3f5efe8a01eca8c2e9d3c32702cf4bead6").save(failOnError: true, validate: false)
 		s1 = new Stream(name: "stream-1")
 		s2 = new Stream(name: "stream-2")
 		s3 = new Stream(name: "stream-3")
+
+		pub1Inbox = new Stream(name: "publisher1", inbox: true)
```

Comment:
Lots of new declarations but no new assertions? 

(This class shouldn't actually even be testing/concerned with anything that happens inside `PermissionService`. Looks like this test is using the real `PermissionService`, while it should be testing against a mock. Easy refactor?)

Reply:
I tried but the refactor is not that easy as most of these tests assert on the creation / update / deletion of `Permission`s instead of asserting calls of a mock `PermissionService`. Since this is prior to this PR and a different issue, I feel like it should be done in a separate PR.